src/validators/ohri-form-validator.test.ts

```diff
@@ -0,0 +1,101 @@
+import { OHRIFormField } from '../api/types';
+import { OHRIFieldValidator } from './ohri-form-validator';
+
+describe('OHRIFieldValidator - validate', () => {
+  const numberInputField: OHRIFormField = {
+    label: 'A Question of type obs that renders a Number input',
+    type: 'obs',
+    questionOptions: {
+      rendering: 'number',
+      concept: 'a-system-defined-concept-uuid',
+      min: '5',
+      max: '10',
```

Comment:
for TextInput i feel having the name `minLength` and `maxLength` is more descriptive of what we want to achieve with it.